Sources/Orbit/Support/Components/TagGroup.swift

```diff
@@ -23,8 +23,8 @@ public struct TagGroup<TM: TagModel>: View {
 
     @Environment(\.accessibilityReduceMotion) private var isReduceMotionEnabled
     @Environment(\.isFadeInEnabled) private var isFadeInEnabled
-    @State private var tagsFadeIn: [Bool] = []
-    @Binding private var tags: [TM]
+    @Binding var tags: [TM]
+    @State var tagsFadeIn: [Bool] = []
```

Comment:
Looks like we have an unwritten convention to order the property wrappers:
- `@Environment`
- `@Binding`
- `@State`

We should probably start a document with those, or at least an issue if we don't have enough of them yet.

Reply:
Currently, it follows the loose order of how exposed the property is. 

It works for now, but maybe we may be breaking it in future, as the `@Binding` property is the one that can actually appear in the synthesized initializer for some internal helper views.